File: 1020-Age-in-Days/1020.go
package main

import "fmt"

func main() {

	var days, period int

	fmt.Scan(&days)
	period = days / 365
	fmt.Printf("%d ano(s)\n", period)
	days = days % 365
	period = days / 30
	fmt.Printf("%d mes(es)\n", period)
	days = days % 30
	fmt.Printf("%d dia(s)\n", days)

}

File: 1020-Age-in-Days/1020.py
days = int(input())

period = days // 365
print("{:d} ano(s)".format(period))
days = days % 365
period = days // 30
print("{:d} mes(es)".format(period))
days = days % 30
print("{:d} dia(s)".format(days))